src/app/home/home.html

```diff
@@ -15,10 +15,10 @@
   </button>
   <button
     md-button
-    [class.highlight]="currentTab === 'commit'"
-    [routerLink]="['/home', { tab: 'commit' }]"
+    [class.highlight]="currentTab === 'ask'"
+    [routerLink]="['/home', { tab: 'ask' }]"
   >
-    COMMIT
+    ASK
```

Comment:
Maybe 'QUESTIONS' would be a better tab title than 'ASK'. (Since the other tab names are nouns as well)